selfdrive/car/honda/interface.py

```diff
@@ -543,10 +543,8 @@ def update(self, c):
     # it can happen that car cruise disables while comma system is enabled: need to
     # keep braking if needed or if the speed is very low
     if self.CP.enableCruise and not ret.cruiseState.enabled and c.actuators.brake <= 0.:
-      # non loud alert if cruise disbales below 25mph as expected (+ a little margin)
+      # send loud alert if cruise disables below 25mph as expected (+ a little margin)
       if ret.vEgo < self.CP.minEnableSpeed + 2.:
-        events.append(create_event('speedTooLow', [ET.IMMEDIATE_DISABLE]))
```

Comment:
I'm ok with the change, but you should change this in the alert database. `speedTooLow` is still what we want to be issued

Reply:
Done.  Restored the commented out code and changed alert.py instead.